helm/alfresco-process-application/values.yaml

```diff
@@ -118,6 +119,8 @@ runtime-bundle:
       value: "{{ .Release.Name }}"
     - name: KEYCLOAK_USERESOURCEROLEMAPPINGS
       value: "true"
+    - name: ACT_RB_SERVICE_URL
```

Comment:
this one is not needed by DMN, but for form-service as next task yes so we might even keep it here. Up to you

Reply:
was wrong the values below